lib/FileApprover.js

```diff
@@ -28,8 +28,19 @@ exports.verify = function (namer, writer, reporterFactory, options) {
     writer.write(approvedFileName);
   }
 
+  var selectFirstCompatibleReporter = function () {
+    var reporter = reporterFactory().filter(function (reporterCandidate) {
+      return reporterCandidate.canReportOn(receivedFileName);
+    })[0];
```

Comment:
@martinsson bit confused how this passed tests and worked for you... `reporterFactory()` seems to be returning the `DiffReporterAggregate` and not an array of reporters. Causing test failures on my windows machine when I `gulp test`... debugging now...